radicle-httpd/src/api/json.rs

```diff
@@ -134,6 +137,24 @@ fn name_in_path(path: &str) -> &str {
     }
 }
 
+fn get_branches(
+    repo: &git::Repository,
+    id: &ActorId,
+    head: &Oid,
+) -> Result<Vec<RefString>, refs::Error> {
+    let remote = repo.remote(id)?;
+    let branches = remote
+        .refs
+        .iter()
+        .filter_map(|(name, o)| match o == head {
+            true => Some(name.to_owned()),
+            false => None,
+        })
```

Comment:
Maybe you can use an `if` instead of the `match`?

Reply:
fixed in e438753

Reply:
Hehe it's a pretty common pattern :shrug: 

There's also https://doc.rust-lang.org/std/primitive.bool.html#method.then_some

Reply:
Oh nice! Will jot that down